includes/internal/student-progress/course-progress/repositories/class-comments-based-course-progress-repository.php

```diff
@@ -47,12 +59,12 @@ public function create( int $course_id, int $user_id ): Course_Progress_Interfac
 			throw new \RuntimeException( "Can't create a course progress" );
 		}
 
-		$progress = $this->get( $course_id, $user_id );
-		if ( ! $progress ) {
+		$comment = get_comment( $comment_id );
```

Comment:
Just curious, any particular reason we removed the usage of the 'get' function and fetched the comment directly here now?

Reply:
Initially, it helped me to partially solve an issue with a case when it can't find a comment right after creating it.
Later, I found it didn't solve it entirely and found another WPML-focused solution, but decided to leave this change as it should be a more optimized version of fetching a comment (it uses the primary key here).